src/Mod/Sketcher/Gui/DrawSketchHandler.cpp

```diff
@@ -885,6 +885,22 @@ void DrawSketchHandler::createAutoConstraints(const std::vector<AutoConstraint>&
     }
 }
 
+int DrawSketchHandler::seekAndRenderAutoConstraint(

```

Comment:
Why make this return an int? I don't see any uses of that value.

Reply:
I've done this because `seekAutoConstraint` was returning an int. So for convenience in case it could be usefull I also returned int here too.
However it seems that `seekAutoConstraint` was returning int in the sole purpose of testing `if seekAutoConstraint() { do something }` which is now not necessary.

So the return int can be removed. Do you want me to remove it ?